examples/Cargo.toml

```diff
@@ -15,25 +15,21 @@ name = "getting_started"
 path = "getting_started.rs"
 
 [[example]]
-name = "account_basic"
-path = "account/basic.rs"
+name = "account_create_did"
+path = "account/create_did.rs"
 
 [[example]]
 name = "account_config"
 path = "account/config.rs"
 
 [[example]]
-name = "account_methods"
-path = "account/methods.rs"
```

Comment:
I think the `account_methods` example needs to be deleted? It still shows up in this branch I think.